phpBB/app.php

```diff
@@ -28,7 +28,9 @@
 $auth->acl($user->data);
 $user->setup('app');
 
+/* @var $http_kernel Symfony\Component\HttpKernel\HttpKernel */
```

Comment:
It depends if the comment is in a namespace or not. I can add a leading slash everywhere to be sure.